genny-project/layouts
# internmatch refactor view detail layouts

https://outcomelife.atlassian.net/browse/INTM-318

## Description:
Refactoring Detail View for Companies, and Contacts.

## How to Test:
1) Click on Companies from Treeview.
2) Click on Detail View.
3) Check information is correct, cross check with store.
4) Repeat for all sub items, Contacts, and first level of Contacts sub items.
